### PR TITLE
Update to return single quotes and tabs instead of spaces

### PR DESCRIPTION
**Pull request by djwester**

### CODE REVIEW

**Comment by djwester** [comment]
the irony of need to disable eslint so that we can fix eslint errors elsewhere.....

File: src/build.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ function buildLang(langName, langData, buildTemplate, expand) {
 		},
 		{
 			regex: /{{langData}}/g,
-			value: expand ? JSON.stringify(langData, null, 4) : JSON.stringify(langData)
+			value: expand ? JSON.stringify(langData, null, '\t\t').replace(/"/g, "'") : JSON.stringify(langData)/* eslint-disable-line quotes */
```
Comment (mpharoah-d2l) [inline src/build.js]: Is it supposed to double tab for each indent?

Comment (mpharoah-d2l) [reply src/build.js]: You can avoid the es-lint-disable with `'\''`

Comment (djwester) [reply src/build.js]: Yeah we need two tabs 
image

**Pull request merged**